api/users/views.py

```diff
@@ -288,25 +287,23 @@ class UserNodes(JSONAPIBaseView, generics.ListAPIView, UserMixin, ODMFilterMixin
     view_category = 'users'
     view_name = 'user-nodes'
 
+    ordering = ('-date_modified')
+
     # overrides ODMFilterMixin
     def get_default_odm_query(self):
         user = self.get_user()
         return (
-            Q('contributors', 'eq', user) &
+            Q('contributors', 'contains', user._id) &
+            Q('is_folder', 'ne', True) &
             Q('is_collection', 'ne', True) &
-            Q('is_deleted', 'ne', True)
+            Q('is_deleted', 'ne', True) &
+            Q('is_registration', 'ne', True)
```

Comment:
Is this a query that can be overriden by clients using the querystring? Or is this a baseline query that clients may only *further* constrain? I ask because we advertise in the docstring that `registration` is filterable, and if the latter then this would seem to be a breaking change.